### ISSUE TITLE
Difficulty creating an Initial Machine

### ISSUE THREAD

**Issue by norwalk350** [Seitanas/kvm-vdi#62]
Thanks again for helping me previously on the .qcow2 issue! 

I'm having another issue with creating an initial machine. I tried to create the machine in the dashboard.

![initial](https://cloud.githubusercontent.com/assets/22620330/19197364/766458f8-8c87-11e6-88db-96f0f2fd0f7d.png)

However, when I hit the Create VM's button, nothing happens. I checked the apache2 error.log file and found this:

![initial_error](https://cloud.githubusercontent.com/assets/22620330/19197395/a18249e6-8c87-11e6-97c8-9b6ad5eec4e3.png)

Again, if this is something I misconfigured, please let me know. I'm very close with this!

**Comment by Seitanas** [comment]
Please update your KVM-VDI and try again

**Comment by norwalk350** [comment]
Another error popped up (T_VARIABLE on line 91). I researched the error and found that it pops up when a semicolon is missing. So, I added the semicolon at the end of line 91. After doing this, the Initial machine looked like it started working (I added a commit to that line). 

So I thought....

After creating the machine, I went into the dashboard and found that it just sits on creation.

![image](https://cloud.githubusercontent.com/assets/22620330/19239090/822130c4-8ed2-11e6-8f78-7e25128280e8.png)

Here is my configuration:

![image](https://cloud.githubusercontent.com/assets/22620330/19239170/d842f118-8ed2-11e6-98c1-500f57dda5c7.png)

Here is the error I get when I create the machine:

![image](https://cloud.githubusercontent.com/assets/22620330/19239399/e6e29a24-8ed3-11e6-82e6-243832d3453e.png)

Any help is much appreciated! Thanks!
